components/brave_shields/browser/ad_block_engine.h

```diff
@@ -42,11 +42,13 @@ class AdBlockEngine : public base::SupportsWeakPtr<AdBlockEngine> {
   using GetDATFileDataResult =
       brave_component_updater::LoadDATFileDataResult<adblock::Engine>;
 
-  AdBlockEngine();
+  explicit AdBlockEngine(bool is_default_engine);
   AdBlockEngine(const AdBlockEngine&) = delete;
   AdBlockEngine& operator=(const AdBlockEngine&) = delete;
   ~AdBlockEngine();
 
+  bool IsDefaultEngine() { return is_default_engine_; }
+
```

Comment:
I guess my intention is to move away from the "default" terminology since it's getting a bit confusing (and since #18829, not a useful distinction either). So, anything we can do towards that end would be nice. But it's not really an issue with this PR specifically.

Reply:
I think these changes are pretty explicitly to make things work with the default engine distinction. FWIW, and maybe this is worth looking at, the `default_exception_filters_provider_` is marked as being part of the default engine (via the default argument to the ctr of `AdBlockComponentFiltersProvider`), while regional and subscription are marked as not being default.